### ISSUE TITLE
Data18 Still problem ??

### ISSUE THREAD

**Issue by pibi73** [PAhelper/PhoenixAdult.bundle#1538]
**Describe the bug**
Data18 Still don't load date and actor name, with the last Phoenix.bundle.

Thanks.

**Include**
1. Scene URL
2. File Name
3. Log File

**Comment by ConstrictM** [comment]
Can you provide this stuff please:

Scene URL
File Name
Log File

**Comment by raedwulf5** [comment]
Scene URL: https://www.data18.com/scenes/4188828

File Name: Data18Scenes - 4188828 - [Scene 2 - Allie Haze in Bound By Desire 3].mp4

Log File: [com.plexapp.agents.phoenixadult.log](https://github.com/PAhelper/PhoenixAdult.bundle/files/8373803/com.plexapp.agents.phoenixadult.log)

**Comment by chempke** [comment]
Sorry to tell, but there are still scenes that won't work with this update. 

Scene URL: [https://www.data18.com/scenes/3110634](url)
File Name: data18scenes - 3110634
Log File: [com.plexapp.agents.phoenixadult.log](https://github.com/PAhelper/PhoenixAdult.bundle/files/8377675/com.plexapp.agents.phoenixadult.log)

no Actors

**Comment by DirtyRacer1337** [comment]
Try with this 6e5fbf26076435cfab819c217c2b9cb571bdab91

**Comment by chempke** [comment]
no sorry, won't work either

```
2022-03-29 23:42:18,764 (7f8e3ff23808) :  DEBUG (agentkit:1132) - Creating new agent class called PhoenixAdultAgent
2022-03-29 23:42:18,764 (7f8e3ff23808) :  DEBUG (agentkit:937) - Updating agent information: [{'media_types': ['Movie'], 'accepts_from': ['com.plexapp.agents.localmedia', 'com.plexapp.agents.lambda'], 'fallback_agent': None, 'contributes_to': ['com.plexapp.agents.stashplexagent', 'com.plexapp.agents.themoviedb', 'com.plexapp.agents.imdb'], 'languages': ['en', 'de', 'fr', 'es', 'it', 'nl'], 'persist_stored_files': True, 'version': 0, 'primary_provider': True, 'prefs': True, 'name': 'PhoenixAdult'}]
2022-03-29 23:42:18,766 (7f8e3ff23808) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0FnZW50U2VydmljZTpVcGRhdGVJbmZv/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQo3CmRpY3QKbGlzdApkaWN0Cmxpc3QKbGlzdApsaXN0Cmxpc3QKMgpzMzEKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdHMxMAppZGVudGlmaWVycjEKczEwCmFnZW50X2luZm8xCnIyCjEwCnIzCnMxMQptZWRpYV90eXBlc3I0CnMxMgphY2NlcHRzX2Zyb21uczE0CmZhbGxiYWNrX2FnZW50cjUKczE0CmNvbnRyaWJ1dGVzX3RvcjYKczkKbGFuZ3VhZ2VzYjFzMjAKcGVyc2lzdF9zdG9yZWRfZmlsZXNpMApzNwp2ZXJzaW9uYjFzMTYKcHJpbWFyeV9wcm92aWRlcmIxczUKcHJlZnNzMTIKUGhvZW5peEFkdWx0czQKbmFtZTEKczUKTW92aWUyCnMyOQpjb20ucGxleGFwcC5hZ2VudHMubG9jYWxtZWRpYXMyNQpjb20ucGxleGFwcC5hZ2VudHMubGFtYmRhMwpzMzMKY29tLnBsZXhhcHAuYWdlbnRzLnN0YXNocGxleGFnZW50czI5CmNvbS5wbGV4YXBwLmFnZW50cy50aGVtb3ZpZWRiczIzCmNvbS5wbGV4YXBwLmFnZW50cy5pbWRiNgpzMgplbnMyCmRlczIKZnJzMgplc3MyCml0czIKbmxyMAo_'
2022-03-29 23:42:18,934 (7f8e3ff23808) :  INFO (core:611) - Started plug-in
2022-03-29 23:42:18,935 (7f8e3ff23808) :  DEBUG (socketinterface:160) - Starting socket server
2022-03-29 23:42:18,936 (7f8e3ff23808) :  DEBUG (runtime:1117) - Created a thread named 'start'
2022-03-29 23:42:18,936 (7f8e3ff23808) :  INFO (socketinterface:184) - Socket server started on port 41840
2022-03-29 23:42:18,936 (7f8e3ff23808) :  INFO (pipeinterface:25) - Entering run loop
2022-03-29 23:42:18,936 (7f8e3ff23808) :  DEBUG (runtime:717) - Handling request GET /:/prefixes
2022-03-29 23:42:18,938 (7f8e3ff23808) :  DEBUG (runtime:814) - Found route matching /:/prefixes
2022-03-29 23:42:19,012 (7f8e3ff23808) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 156 bytes
2022-03-29 23:42:19,505 (7f8e3a0e3b38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmRlczQKbGFuZ2IwczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgowMDExZTA5ODZjMjdhNDUwczE3Cm9wZW5TdWJ0aXRsZXNIYXNoczIwCkRhdGExOHNjZW5lcyA1Mzc4ODQ5czQKbmFtZXMxODgKJTJGdm9sdW1lMSUyRk5BUyUyRjQlMkUlMjBWaWRlbyUyRjMlMkUlMjBYWFglMkYyJTJFJTIwUGVyZm9ybWVyJTJGQWolMjBBcHBsZWdhdGUlMkZDYWwlMjBWaXN0YSUyRlJlZG5lY2slMjBHaXJscyUyMFRyYWlsZXIlMjBUcmFzaCUyMEhvZXMlMjAtJTIwU2NlbmUlMjAzJTJGZGF0YTE4c2NlbmVzJTIwLSUyMDUzNzg4NDklMkVtcDRzOApmaWxlbmFtZXM0MAo5NzZlMGRkYWM4Njc0NDU0NTZlMDJhOWJiYWM0MjA3ODAyOTgwODk4czgKcGxleEhhc2hzMgotMXM4CmR1cmF0aW9uczYKMjgxMzY3czIKaWRyMAo_
2022-03-29 23:42:19,506 (7f8e3a0e3b38) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2022-03-29 23:42:19,524 (7f8e3a0e3b38) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmRlczQKbGFuZ2IwczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgowMDExZTA5ODZjMjdhNDUwczE3Cm9wZW5TdWJ0aXRsZXNIYXNoczIwCkRhdGExOHNjZW5lcyA1Mzc4ODQ5czQKbmFtZXMxODgKJTJGdm9sdW1lMSUyRk5BUyUyRjQlMkUlMjBWaWRlbyUyRjMlMkUlMjBYWFglMkYyJTJFJTIwUGVyZm9ybWVyJTJGQWolMjBBcHBsZWdhdGUlMkZDYWwlMjBWaXN0YSUyRlJlZG5lY2slMjBHaXJscyUyMFRyYWlsZXIlMjBUcmFzaCUyMEhvZXMlMjAtJTIwU2NlbmUlMjAzJTJGZGF0YTE4c2NlbmVzJTIwLSUyMDUzNzg4NDklMkVtcDRzOApmaWxlbmFtZXM0MAo5NzZlMGRkYWM4Njc0NDU0NTZlMDJhOWJiYWM0MjA3ODAyOTgwODk4czgKcGxleEhhc2hzMgotMXM4CmR1cmF0aW9uczYKMjgxMzY3czIKaWRyMAo_
2022-03-29 23:42:19,524 (7f8e3a0e3b38) :  INFO (agentkit:961) - Searching for matches for {'openSubtitlesHash': '0011e0986c27a450', 'name': 'Data18scenes 5378849', 'filename': '%2Fvolume1%2FNAS%2F4%2E%20Video%2F3%2E%20XXX%2F2%2E%20Performer%2FAj%20Applegate%2FCal%20Vista%2FRedneck%20Girls%20Trailer%20Trash%20Hoes%20-%20Scene%203%2Fdata18scenes%20-%205378849%2Emp4', 'plexHash': '976e0ddac867445456e02a9bbac4207802980898', 'duration': '-1', 'id': '281367'}
2022-03-29 23:42:19,525 (7f8e3a0e3b38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/281367/tree'
2022-03-29 23:42:20,091 (7f8e3a0e3b38) :  DEBUG (preferences:258) - Loaded preferences from DefaultPrefs.json
2022-03-29 23:42:20,279 (7f8e3a0e3b38) :  DEBUG (preferences:178) - Loaded the user preferences for com.plexapp.agents.phoenixadult
2022-03-29 23:42:20,281 (7f8e3a0e3b38) :  INFO (__init__:61) - ***MEDIA TITLE [from media.name]*** Data18scenes 5378849
2022-03-29 23:42:20,282 (7f8e3a0e3b38) :  INFO (PAsearchSites:51) - mediaTitle w/ possible abbreviation: Data18scenes 5378849
2022-03-29 23:42:20,314 (7f8e3a0e3b38) :  INFO (PAsearchSites:59) - mediaTitle w/ possible abbrieviation fixed: Data18scenes 5378849
2022-03-29 23:42:20,327 (7f8e3a0e3b38) :  INFO (PAsearchSites:71) - ^^^^^^^ siteNum: 1071
2022-03-29 23:42:20,327 (7f8e3a0e3b38) :  INFO (PAsearchSites:72) - ^^^^^^^ Shortening Title
2022-03-29 23:42:20,331 (7f8e3a0e3b38) :  INFO (PAsearchSites:96) - Search Title (before date processing): 5378849
2022-03-29 23:42:20,331 (7f8e3a0e3b38) :  INFO (__init__:63) - {'siteName': 'data18scenes', 'searchDate': None, 'siteNum': 1071, 'searchTitle': '5378849'}
2022-03-29 23:42:20,332 (7f8e3a0e3b38) :  INFO (PAsearchData:23) - SearchData.title: 5378849
2022-03-29 23:42:20,332 (7f8e3a0e3b38) :  INFO (PAsearchData:29) - SearchData.filename: data18scenes - 5378849
2022-03-29 23:42:20,333 (7f8e3a0e3b38) :  INFO (__init__:100) - Provider: siteData18Scenes
2022-03-29 23:42:21,008 (7f8e3a0e3b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/sys/search.php?index=&key=&key2=&next=1&page=0"
2022-03-29 23:42:22,401 (7f8e3a0e3b38) :  INFO (PAutils:281) - GZip request saved as "bdce333e2bf2486090146bb93775ec9c.gz"
2022-03-29 23:42:22,416 (7f8e3a0e3b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/scenes/5378849"
2022-03-29 23:42:22,820 (7f8e3a0e3b38) :  INFO (PAutils:281) - GZip request saved as "19ad9fa6bf0b415f96e88a39a043fcfe.gz"
2022-03-29 23:42:22,849 (7f8e3a0e3b38) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2022-03-29 23:42:22,849 (7f8e3a0e3b38) :  DEBUG (runtime:924) - Response: [200] str, 764 bytes
2022-03-29 23:42:28,132 (7f8e3b271b38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZGVzNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyODEzNjdzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzOTkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovLzRXM2kyaHlySlRWZ0h2d2d2dDcyMXl6dGlraVp6ejdyNDRMMmdEVUZzbTZ2WmlreGYzNnwxMDcxfD9sYW5nPWRlczQKZ3VpZHM1Nwo0VzNpMmh5ckpUVmdIdndndnQ3MjF5enRpa2laeno3cjQ0TDJnRFVGc202dlppa3hmMzZ8MTA3MXxzMgppZHIwCg__
2022-03-29 23:42:28,133 (7f8e3b271b38) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2022-03-29 23:42:28,134 (7f8e3b271b38) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZGVzNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyODEzNjdzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzOTkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovLzRXM2kyaHlySlRWZ0h2d2d2dDcyMXl6dGlraVp6ejdyNDRMMmdEVUZzbTZ2WmlreGYzNnwxMDcxfD9sYW5nPWRlczQKZ3VpZHM1Nwo0VzNpMmh5ckpUVmdIdndndnQ3MjF5enRpa2laeno3cjQ0TDJnRFVGc202dlppa3hmMzZ8MTA3MXxzMgppZHIwCg__
2022-03-29 23:42:28,135 (7f8e3b271b38) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://4W3i2hyrJTVgHvwgvt721yztikiZzz7r44L2gDUFsm6vZikxf36|1071|?lang=de
2022-03-29 23:42:28,193 (7f8e3b271b38) :  ERROR (model:205) - Cannot read model from /var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Metadata/Movies/b/945d07f601783af12e15ee291fb8a5e7af50db9.bundle/Contents/com.plexapp.agents.phoenixadult
2022-03-29 23:42:28,194 (7f8e3b271b38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/281367/tree'
2022-03-29 23:42:28,206 (7f8e3b271b38) :  INFO (__init__:129) - ******UPDATE CALLED*******
2022-03-29 23:42:28,207 (7f8e3b271b38) :  INFO (__init__:133) - SiteNum: 1071
2022-03-29 23:42:28,207 (7f8e3b271b38) :  INFO (__init__:138) - Provider: siteData18Scenes
2022-03-29 23:42:28,210 (7f8e3b271b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/scenes/5378849"
2022-03-29 23:42:28,426 (7f8e3b271b38) :  INFO (PAutils:281) - GZip request saved as "2b5f8765c52a4fbc8f2a8a98e5b402a6.gz"
2022-03-29 23:42:28,689 (7f8e3b271b38) :  CRITICAL (agentkit:1095) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://4W3i2hyrJTVgHvwgvt721yztikiZzz7r44L2gDUFsm6vZikxf36|1071|?lang=de' (most recent call last):
  File "/volume1/@appstore/PlexMediaServer/Resources/Plug-ins-980a13e02/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1093, in _update
    agent.update(obj, media, lang, **kwargs)
  File "/var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 139, in update
    provider.update(metadata, lang, siteNum, movieGenres, movieActors, valid_images)
  File "/var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/siteData18Scenes.py", line 193, in update
    date = sceneDate if sceneDate else detailsPageElements.xpath('//span[contains(., "Release date")]//following-sibling::a/b')[0].text_content().strip()
  File "/volume1/@appstore/PlexMediaServer/Resources/Plug-ins-980a13e02/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2022-03-29 23:42:29,011 (7f8e3b271b38) :  DEBUG (model:229) - Serializing to /var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Metadata/Movies/b/945d07f601783af12e15ee291fb8a5e7af50db9.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2022-03-29 23:42:29,014 (7f8e3b271b38) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2022-03-29 23:42:29,014 (7f8e3b271b38) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

**Comment by pibi73** [comment]
Same problem here

Scene URL : https://www.data18.com/scenes/13702
Filename : data18scenes - 13702 or data18moviescene - 13702

Load correctly the Studios but don't load Actor.

Best Regards
